app/js/actions/draftActions.js

```diff
@@ -0,0 +1,25 @@
+import { TOGGLE_DRAFT_LAB_ORDER_URGENCY } from './actionTypes';
+import constants from '../utils/constants';
+
+const toggleDraftLabOrderUrgency = (order) => {
+  const orderUuid = order.uuid;
+  let orderUrgency;
+  const hasUrgencyProperty = Object.prototype.hasOwnProperty.call(order, 'urgency');
+  if (hasUrgencyProperty && order.urgency === constants.ROUTINE) {
+    orderUrgency = constants.STAT;
+  } else if (hasUrgencyProperty && order.urgency === constants.STAT) {
+    orderUrgency = constants.ROUTINE;
+  } else {
+    orderUrgency = constants.STAT;
+  }
```

Comment:
since you are basically toggling between two values, I think the conditional blocks you are using is a bit of an overkill. please refactor to something like this.

```
let orderUrgency = constants.STAT
if (hasUrgency) {
   orderUrgency = constants[order.urgency]
}
```

Reply:
Hi @Efosaok 
Yes, you are right, I am toggling between two values, however, the next value actually depends on the previous and that is why the series of checks is needed.
Let me try and explain the checks in plain English:
- If the order has urgency property and its previous value is ROUTINE, the urgency should become STAT.
- If the order has urgency property and its previous value is STAT, the urgency should become ROUTINE.
- If the order does not have urgency property, the urgency should become STAT.

I hope this clarifies your concern.

Reply:
I understand exactly what you are trying to do. but I think what I missed out is that you should set the `constant` variable to this.

```
export const STAT = 'ROUTINE'
export const ROUTINE = 'STAT'
```
and then just use the code up there to replace what you have already. this would make your  code more concise..

Reply:
Hmm..., now I get what you are saying.
I have updated the code with an implementation that reflects what you said above.
Thanks man, I was able to reduce some line of codes(and thinking effort) over there.